controller/src/beerocks/bml/bml_defs.h

```diff
@@ -610,6 +613,42 @@ struct BML_NEIGHBOR_AP {
     uint32_t ap_ChannelUtilization;
 };
 
+struct BML_CLIENT_CONFIG {
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_selected_device;
+    // comma-separated selected bands [2.4,5H,5L,6].
+    char selected_bands[BML_SELECTED_BANDS_LEN];
+    // Optional parameter
+    // The timelife delay of the client configuration (if configured), otherwise 0.
+    uint32_t time_life_delay_days;
+};
+
+struct BML_CLIENT {
+    // Client MAC
+    char sta_mac[BML_MAC_ADDR_LEN];
+    // Time of last client update (in Seconds)
+    uint32_t timestamp_sec;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_selected_device;
+    // comma-separated selected bands [2.4,5H,5L,6].
+    char selected_bands[BML_SELECTED_BANDS_LEN];
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int support_11k;
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int support_11v;
+    // 1 for true, 0 for false, -1 for "parameter is not configured".
+    int support_steering;
```

Comment:
These parameters should be only read only for the user. 
IMO should represent  responsiveness [%]

```suggestion
    // client 11k responsiveness ratio in % since beginning of lifetime configuration".
    int support_11k_responsiveness;
    // client 11v responsiveness ratio in % since beginning of lifetime configuration".
    int support_11v_responsiveness;
    // client steering responsiveness ratio in % since beginning of lifetime configuration".
    int support_steering_responsiveness;
```

Reply:
Why we need all those parameters here?

Reply:
so user could  get client responsiveness (accumulated since first connection) statistics

Reply:
@adam1985d WDYT?

Reply:
these parameters were mentioned when the client's behavior was described in the requirements.
if they are not needed (or they should be provided in some other form like responsiveness) - we can remove/change accordingly - but thought this client **info** would be beneficial for the user

Reply:
These parameters are currently not used and are unneeded as these APIs are only the infrastructure phase, when we decide on the actual values we can expand as needed.
So for now these are removed